### ISSUE TITLE
Format tag name [tag name] by [your company name]

### ISSUE THREAD

**Issue by philpott** [mbaersch/clarity-events#1]
Please update your tag's name to match the format of [tag name] by [your company name] since this is not your own company's tag template.

Once we see this change we'll approve and sync in your template

Thanks!

**Comment by mbaersch** [comment]
Hi!

I am a bit confused: can you tell me what exactly needs to be changed? Would a name like *Microsoft Clarity Events by mbaersch* or *Microsoft Clarity Events by Markus Baersch* instead be correct? 

I am unsure because I did not need to add that to the tag`s name for my *Microsoft UET Consent Mode" template and looking at the alternative Clarity tag from https://github.com/luratic/Clarity which is part of the gallery, too. 

However, I can change the name to whatever is required here, just let me know how exactly it should look like? 

best,
Markus

**Comment by philpott** [comment]
TL;DR either "Microsoft Clarity Events by mbaersch" or "Microsoft Clarity Events by Markus Baersch" would be correct depending on what you want your brand/identifer to be.

This is a recent change that we've implemented on the review process and is being propigated out. So older tags have not had that requirement but we would love for them to be updated to be in that format.

**Comment by mbaersch** [comment]
Thanks for the quick reply. 

Understood... I just changed the name and added "by Markus Baersch" to the name. SHA for the initial release in yaml file got updated, too.  

Allow me to ask another question: would adding parentheses around this suffix be a problem? I would prefer to emphasize my name less, if possible.
